src/utils/throwable.js

```diff
@@ -1,9 +1,16 @@
-// some popup with error message
-
-export async function throwable(callback) {
+import { Popup } from "@/components";
+// todo: should we add customable message here?
```

Comment:
not sure if we need it, but I think it would be useful to be able to customise the message